# Default configuration for emergency handler
---
/**:
    ros__parameters:
      update_rate: 10
      heartbeat_timeout: 0.5
      use_parking_after_stopped: false
